http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/NettySdkHttpClientFactory.java

```diff
@@ -119,6 +130,12 @@ public static Builder builder() {
         return new DefaultBuilder(AttributeMap.builder());
     }
 
+    private Duration isWholeSecond(Duration duration, String param) {
```

Comment:
can we rename this method? `isWholeSecond` feels like it's returning a boolean value.

Reply:
sure - how about `validateIsWholeSecond` ?

Reply:
👍